docs/deployment/manifest-format.md

```diff
@@ -826,46 +830,100 @@ Example manifest:
-| Azure Storage Blobs | `azure.storage.blob.v0` | <span aria-hidden="true">✔️</span><span class="visually-hidden">Yes</span> |
-| Azure Storage Queues | `azure.storage.queue.v0` | <span aria-hidden="true">✔️</span><span class="visually-hidden">Yes</span> |
-| Azure Storage Tables | `azure.storage.table.v0` | <span aria-hidden="true">✔️</span><span class="visually-hidden">Yes</span> |
-| Azure Storage | `azure.storage.v0` | <span aria-hidden="true">✔️</span><span class="visually-hidden">Yes</span> |
-| Container | `container.v0` | <span aria-hidden="true">❌</span><span class="visually-hidden">No</span> |
-| Dockerfile | `docker.v0` | <span aria-hidden="true">❌</span><span class="visually-hidden">No</span> |
-| Postgres Connection | `postgres.connection.v0` | <span aria-hidden="true">✔️</span><span class="visually-hidden">Yes</span> |
-| Postgres Database | `postgres.database.v0` | <span aria-hidden="true">✔️</span><span class="visually-hidden">Yes</span> |
-| Postgres Server | `postgres.server.v0` | <span aria-hidden="true">✔️</span><span class="visually-hidden">Yes</span> |
-| .NET Project | `project.v0` | <span aria-hidden="true">✔️</span><span class="visually-hidden">Yes</span> |
-| RabbitMQ Connection | `rabbitmq.connection.v0` | <span aria-hidden="true">✔️</span><span class="visually-hidden">Yes</span> |
-| RabbitMQ Server | `rabbitmq.server.v0` | <span aria-hidden="true">❌</span><span class="visually-hidden">No</span> |
-| Redis | `redis.v0` | <span aria-hidden="true">✔️</span><span class="visually-hidden">Yes</span> |
-| SQL Server Connection | `sqlserver.connection.v0` | <span aria-hidden="true">❌</span><span class="visually-hidden">No</span> |
-| SQL Server Database | `sqlserver.database.v0` | <span aria-hidden="true">❌</span><span class="visually-hidden">No</span> |
-| SQL Server Server | `sqlserver.server.v0` | <span aria-hidden="true">❌</span><span class="visually-hidden">No</span> |
-
-For more information, see [Deploy a .NET Aspire app to Azure Container Apps using the Azure Developer CLI (in-depth guide)](azure/aca-deployment-azd-in-depth.md).
+The [Azure Developer CLI](/azure/developer/azure-developer-cli/) (azd) is a tool that can be used to deploy .NET Aspire applications to Azure Container Apps. With the `azure.bicep.v0` resource type, cloud-agnostic resource container types can be mapped to Azure-specific resources. The following table lists the resource types that are supported in the Azure Developer CLI:
+
+| Name | Cloud-agnostic API | Configure as Azure resource |
+| ---- | ---------------------------- | ---------------------------------- |
+| Redis | <xref:Aspire.Hosting.RedisBuilderExtensions.AddRedis%2A> | `AsAzureRedis` |
+| Postgres | <xref:Aspire.Hosting.PostgresBuilderExtensions.AddPostgres%2A> | `AsAzurePostgresFlexibleServer` |
+| SQL Server | <xref:Aspire.Hosting.SqlServerBuilderExtensions.AddSqlServer%2A> | `AsAzureSqlDatabase` |
```

Comment:
I think we should default to showing PublishAs instead of As

Reply:
Are there two APIs? What's the difference, and as of the preview-4 bits - I don't recall seeing the `PublishAs` APIs.

Reply:
There are 2 APIs yes. PublishAs means run this as a container locally and publish it as a managed service. As will just break at runtime until we're done with azure provisioning in dev (preview 5)